src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -343,6 +356,8 @@ def submit(self, jobs, info=None):
         # If we're here, then we have submitter components
         self.scriptFile = self.config.JobSubmitter.submitScript
         self.submitDir = self.config.JobSubmitter.submitDir
+        logging.info("ALAN self.submitDir %s and self.config.JobSubmitter.submitDir %s", self.submitDir,
```

Comment:
LoL. I couldn't find where we set this var (and it's not in the config), that's why I wanted to check on it :-)